src/main/java/com/zebrunner/agent/junit5/JUnit5Adapter.java

```diff
@@ -45,14 +45,18 @@ public void registerRunFinish(TestPlan testPlan) {
 
     public void registerTestStart(TestIdentifier testIdentifier) {
         if (testIdentifier.isTest()) {
-            OffsetDateTime startedAt = OffsetDateTime.now();
+
             String uuid = testIdentifier.getUniqueId();
             List<UniqueId.Segment> idSegments = UniqueId.parse(uuid).getSegments();
             MethodSource source = (MethodSource) testIdentifier.getSource().get(); // additional validation here?
             boolean dynamic = SegmentResolver.isDynamic(idSegments);
             boolean parameterized = SegmentResolver.isParameterizedTest(idSegments);
             String testName = buildTestName(testIdentifier, dynamic || parameterized);
 
+/*
+* TODO: should this commented out code be deleted?
+*  If this is some form of comment, should it be javadoc?
+*/
```

Comment:
Don't remember the actual purpose of the commented out code. Let's keep it as is